src/components/pages/Login.js

```diff
@@ -3,6 +3,8 @@ import { useState } from "react";
 import { Redirect } from "react-router-dom";
 import { loginHandler } from "../../handlers/authenticationHandlers.js";
 
+import { URL } from './../../utils/generalVariables'
```

Comment:
💡 Not sure here, but I though we'd be using `URL` in the "path" variable for each request handler. And then we just import that "path" variable. 

For example:
```
paths = {
    login: ${URL}/login
}
...
import { login} ...

...

axios.post(login, ...)
```
But I might be wrong here.

Reply:
You are correct. It's an import I forgot to remove after some testing and tweaking. Will remove!